### DISCUSSION TITLE
How To Unblock anything!1!!!!1

### DISCUSSION THREAD

**Discussion by Bunjolio** [3kh0/ext-remover#409] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Step 1: get yo money up not yo funny up
Step 2: get your own laptop/chromebook
step 3: bring to school
step 4: profit

**Comment by 3kh0** [comment, marked as answer]
or run kasm

**Reply by rwg100** [reply]
or run racism

**Reply by Bunjolio** [reply]
Racism method: hate black people👍

**Reply by Bunjolio** [reply]
> or run kasm
wat is that

**Reply by Aka-but-nice** [reply]
Kasm is a linux type device
